server/routes/testutils/appSetup.ts

```diff
@@ -45,6 +49,10 @@ function appSetup(services: Services, production: boolean, userSupplier: () => E
     }
     next()
   })
+  app.use((req, res, next) => {
+    req.id = uuidv4()
```

Comment:
not a comment for this PR, but we should probably add this as a custom dimension in app insights